### ISSUE TITLE
Pledge without contribution in Online Contrib Statement Error

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#2199]
### Prerequisites
* [x] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

The Online (Lava) Contribution Statement throws an exception if a person has a pledge, but there are not yet any contributions to that fund/pledge.

### Steps to Reproduce

1. Add a pledge to a fund for someone (but with NO contributions to that fund, yet)
2. Generate an online contrib statement for that someone
3. It blows up

**Expected behavior:** 

Statement would generate without error.

**Actual behavior:** 

Throws exception. Daniel H says it's a problem in ContributionStatementLava.ascx.cs not returning a default of 0 when there are no contributions to the pledged fund.

![image](https://cloud.githubusercontent.com/assets/3903408/26068526/6c2becbe-3963-11e7-9453-aa332fa032e2.png)

### Versions

* **Rock Version:** 6.2

**Comment by cabal95** [comment]
The general fix (and a good coding standard to use) would be instead of:
```
    .Sum(t => t.Amount)
```
using:
```
    .Select( t => t.Amount)
    .DefaultIfEmpty( 0 )
    .Sum();
````

Sum and related LINQ functions blow up if the result set is empty.

**Comment by garrettjohnson** [comment]
Fixed via 932e9e0
